src/server/verification/verificationAPI.js

```diff
@@ -456,20 +461,11 @@ const setup = (app: Router, verifier: VerificationAPI, storage: StorageAPI) => {
           })
 
           release()
-
-          return res.status(200).json({
-            ok: 1
-          })
         },
         onError: e => {
           log.error('Bonuses charge failed', e.message, e, user)
 
           fail()
```

Comment:
maybe check if connection is still open and send json result if it is, maybe ontxhash wont be called